site/docs/plugins/README.md

```diff
@@ -33,23 +33,26 @@ Installing plugins is fun and easy, and we want you to know what we have in stor
+| [Sessions](./session.md)                      | _built-in_                                            | Store user data in your database                     |
+| [Inline and Custom Keyboards](./keyboard.md)  | _built-in_                                            | Simplify building inline and custom keyboards        |
+| [Media Groups](./media-group.md)              | _built-in_                                            | Simplify sending media groups and editing media      |
+| [Inline Queries](./inline-query.md)           | _built-in_                                            | Easily build results for inline queries              |
+| [Auto-retry](./auto-retry.md)                 | [`auto-retry`](./auto-retry.md)                       | Automatically handle rate limiting                   |
+| [Conversations](./conversations.md)           | [`conversations`](./conversations.md)                 | Build powerful conversational interfaces and dialogs |
+| [Chat Members](./chat-members.md)             | [`chat-members`](./chat-members.md)                   | Track which user joined which chat                   |
+| [Emoji](./emoji.md)                           | [`emoji`](./emoji.md)                                 | Simplify using emoji in code                         |
+| [Files](./files.md)                           | [`files`](./files.md)                                 | Easily handle files                                  |
+| [Hydration](./hydrate.md)                     | [`hydrate`](./hydrate.md)                             | Call methods on objects returned from API calls      |
+| [Internationalization](./i18n.md)             | [`i18n`](./i18n.md) or [`fluent`](./fluent.md)        | Let your bot speak multiple languages                |
+| [Interactive Menus](./menu.md)                | [`menu`](./menu.md)                                   | Design dynamic button menus with flexible navigation |
+| [Parse Mode](./parse-mode.md)                 | [`parse-mode`](./parse-mode.md)                       | Simplify message formatting                          |
+| [Rate Limiter](./ratelimiter.md)              | [`ratelimiter`](./ratelimiter.md)                     | Automatically restrict users who spam your bot       |
+| [Router](./router.md)                         | [`router`](./router.md)                               | Route messages to different parts of your code       |
+| [Runner](./runner.md)                         | [`runner`](./runner.md)                               | Perform long polling concurrently and at scale       |
+| [Stateless Question](./stateless-question.md) | [`stateless-question`](./stateless-question.md)       | Create dialogs without a data storage                |
+| [Throttler](./transformer-throttler.md)       | [`transformer-throttler`](./transformer-throttler.md) | Slow down API calls                                  |
```

Comment:
Why duplicate links? Wouldn't it be easier to add a link to `built-in` or remove the second column, and move `built-in` and package names to the first one in brackets next to the plugin name? From the user's point of view, I can think that the links in the first column lead to the documentation page, and the second column to the NPM

Reply:
We can remove the links from the second column, and adjust the text above.

Initially, we added the links to packages because of the i18n row which requires two links. However, I find it very counter-intuitive that the plugin names cannot be clicked. In addition, the package name links have poor visibility due to the `code` formatting.

Reply:
If we remove the links from the packages, we need to find a solution for i18n. I have none (apart from just not linking to the fluent plugin anymore).

Reply:
`Internationalisation: [\`i18n\`](./i18n.md) or [\`fluent\`](./fluent.md)`

Reply:
I had a conversation with @deptyped that the fluent plugin could be deprecated (just leave a note on the page that it is better to use i18n)

Reply:
We already have such a remark. It is not necessary to deprecate plugins that are used and maintained. If it is too hard to decide between the plugins, we can improve the remark.